autogpt_platform/backend/backend/data/block_cost_config.py

```diff
@@ -376,6 +383,54 @@
             },
         )
     ],
+    EnrichlayerLinkedinProfileBlock: [
+        BlockCost(
+            cost_amount=1,
+            cost_filter={
+                "credentials": {
+                    "id": enrichlayer_credentials.id,
+                    "provider": enrichlayer_credentials.provider,
+                    "type": enrichlayer_credentials.type,
+                }
+            },
+        )
+    ],
+    EnrichlayerLinkedinPersonLookupBlock: [
+        BlockCost(
+            cost_amount=2,
+            cost_filter={
+                "credentials": {
+                    "id": enrichlayer_credentials.id,
+                    "provider": enrichlayer_credentials.provider,
+                    "type": enrichlayer_credentials.type,
+                }
+            },
+        )
+    ],
+    EnrichlayerLinkedinRoleLookupBlock: [
+        BlockCost(
+            cost_amount=3,
+            cost_filter={
+                "credentials": {
+                    "id": enrichlayer_credentials.id,
+                    "provider": enrichlayer_credentials.provider,
+                    "type": enrichlayer_credentials.type,
+                }
+            },
+        )
+    ],
+    EnrichlayerLinkedinProfilePictureBlock: [
+        BlockCost(
+            cost_amount=3,
+            cost_filter={
+                "credentials": {
+                    "id": enrichlayer_credentials.id,
+                    "provider": enrichlayer_credentials.provider,
+                    "type": enrichlayer_credentials.type,
+                }
+            },
+        )
+    ],
```

Comment:
Is @Swiftyos's block SDK working already? Or do we still need to do this on a seperate file

Reply:
its working but this is from before that (i know crazy)